### PR TITLE
[INSIGHTS-31] Add extra detail to Algorithmia Insights documentation

### PR DESCRIPTION
**Pull request by dinoboy197**
Adding several updates to documentation for Insights:
* Rewording some awkward language
* Adding unencrypted connection configuration and clarifying SCRAM connection configuration
* Added screenshots of configuration for unencrypted and SCRAM configurations
* Added info about reserved metric names that must not be used by client code
* Added info about best-effort approach to send metrics to customer's Kafka broker before dropping them
* Added info about user-created metrics maximums
* Added screenshots for publishing with insights enabled and viewing algorithms with insights enabled flag

Please see the Insights documentation on our testing server to view once rendered: https://insights1.enthalpy.click/developers/algorithmia-enterprise/insights

### CODE REVIEW

File: _pages/algorithmia-enterprise/insights.md [hunk 1]
```diff
@@ -42,13 +66,28 @@ Each algorithm in your Algorithmia cluster has independent settings for whether
...
+* `algorithm_version`
+
 ### Enabling Algorithmia Insights for an algorithm version
 
 Enabling Algorithmia Insights for an algorithm version happens during [algorithm publishing](../algorithm-development/your-first-algo/#publish-your-algorithm).
 
 When using the web UI publishing workflow, a toggle will appear to enable Algorithmia Insights for the specific algorithm version that you are publishing.
 
-**Screenshot pending UX completion - https://algorithmia.atlassian.net/browse/INSIGHTS-31**
+![](/developers/images/algorithmia-enterprise/insights/web-ui-publish.png)
```
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Missing alt description.

File: _pages/algorithmia-enterprise/insights.md [hunk 2]
```diff
@@ -62,4 +101,4 @@ Once you've published your algorithm with Algorithmia Insights enabled, all exec
 
 The "Versions" tab of the algorithm page shows which published versions of that algorithm have Algorithmia Insights enabled:
 
-**Screenshot pending UX completion - https://algorithmia.atlassian.net/browse/INSIGHTS-31**
+![](/developers/images/algorithmia-enterprise/insights/web-ui-versions.png)
```
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Missing alt description.

**Review by eliperelman — APPROVED** [review]
LGTM! Just have a couple nits for some image alt descriptions.

File: _pages/algorithmia-enterprise/insights.md (hunk 1 above)
Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: Thanks! Done.

File: _pages/algorithmia-enterprise/insights.md (hunk 2 above)
Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: Done

**Pull request merged**